iidm/iidm-xml-converter/src/main/resources/xsd/generatorEntsoeCategory.xsd

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema version="1.0"
+           xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           targetNamespace="http://www.itesla_project.eu/schema/iidm/ext/generator_entsoe_category/1_0"
+           elementFormDefault="qualified">
+    <xs:element name="entsoeCategory">
+        <xs:simpleType>
+            <xs:restriction base="xs:integer">
+                <xs:minInclusive value="1"/>
+                <xs:maxInclusive value="42"/>
```

Comment:
I think we should remove this upper bound check, because they very often add new values and it would be painful and useless to upgrade the extension just for that.